support-workers/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "support-workers",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --group=-integration",
+    "it-test": "jest --group=integration",
+    "lint": "eslint src/typescript/**/*.ts",
+    "check-formatting": "prettier --check **.ts",
+    "build": "tsc",
+    "clean-target": "rm -rf target && mkdir -p target",
+    "install-runtime-dependencies": "yarn install --modules-folder target/typescript/node_modules --production",
+    "build-and-zip": "yarn clean-target && yarn install-runtime-dependencies && yarn build && cd target/typescript && zip -r support-workers.zip ./* -x \".*\" -x \"__MACOSX\"",
```

Comment:
why do we need to clean first, is it just trying to be on the safe side in case incremental build causes an issue?

Reply:
So that we make sure we only get what's really needed in the zip, not any cruft that's built up during development. Not crucial for pre-prod but that was the thinking